mne/viz/tests/test_3d.py

```diff
@@ -494,11 +494,14 @@ def test_plot_dipole_mri_orthoview():
     """Test mpl dipole plotting."""
     dipoles = read_dipole(dip_fname)
     trans = read_trans(trans_fname)
-    for coord_frame, idx, show_all in zip(['head', 'mri'],
-                                          ['gof', 'amplitude'], [True, False]):
+    for coord_frame, idx, show_all, title in zip(['head', 'mri'],
+                                                 ['gof', 'amplitude'],
+                                                 [True, False],
+                                                 ['Test', None]):
```

Comment:
now would be a good time to change this to a pytest parametrize approach, if you're up for it.

Reply:
Definitely, thanks for your suggestion.

Reply:
Addressed in 382586b